python/cudf_polars/cudf_polars/utils/config.py

```diff
@@ -128,6 +128,8 @@ def validate(config: dict) -> None:
                 "parquet_blocksize",
                 "cardinality_factor",
                 "groupby_n_ary",
+                "bcast_join_limit",
```

Comment:
Should we spell the out as `broadcast_join_limit`?

Reply:
Sure - I changed it to "broadcast_join_limit"